dj/models/node.py

```diff
@@ -82,15 +110,25 @@ class NodeYAML(TypedDict, total=False):
     tables: Dict[str, List[TableYAML]]
 
 
-class Node(SQLModel, table=True):  # type: ignore
+class NodeBase(SQLModel):
```

Comment:
I assume you considered `BaseNode` and decided to call it `NodeBase` ?

Reply:
I hadn't considered `BaseNode` but I'm not entirely opposed to it. Although at first glance it might imply that there's a node type "base".

Reply:
I see... makes sense. NodeBase it is.